src/web/js/dashboard/config.js

```diff
@@ -0,0 +1,9 @@
+// App name in *Title Case*.
+export const APP_NAME = 'Pyret';
```

Comment:
I guess these should probably be centralized into the .env file (or some other central config location), right?